Not working on AMD64

Hi 
Thanks for the code , but the code is  not working on AMD64 , by throwing a error

cmd/certs_storage.go:200:28: wrapperFunc: use strings.ReplaceAll method in `strings.Replace(domain, "*", "_", -1)` (gocritic)
	safe, err := idna.ToASCII(strings.Replace(domain, "*", "_", -1))

--- a/cmd/certs_storage.go
+++ b/cmd/certs_storage.go
@@ -197,7 +197,7 @@ func (s *CertificatesStorage) MoveToArchive(domain string) error {
 
 // sanitizedDomain Make sure no funny chars are in the cert names (like wildcards ;)).
 func sanitizedDomain(domain string) string {
-	safe, err := idna.ToASCII(strings.Replace(domain, "*", "_", -1))
+	safe, err := idna.ToASCII(strings.ReplaceAll(domain, "*", "_"))
 	if err != nil {
 		log.Fatal(err)
 	}

